exercises/atbash-cipher/tests/atbash-cipher.rs

```diff
@@ -0,0 +1,70 @@
+extern crate atbash_cipher as cipher;
+
+#[test]
+fn test_encode_yes() {
+    assert_eq!("bvh", cipher::encode("yes"));
+}
+
+#[test]
+fn test_encode_no() {
+    assert_eq!("ml", cipher::encode("no"));
+}
+
+#[test]
+fn test_encode_OMG() {
```

Comment:
Rust gives a warning about this name. It would prefer `test_encode_omg`, which is fine.